tests/test_data/allrecipes.com/allrecipescurated.json

```diff
@@ -26,23 +26,23 @@
   "category": "Dinner",
   "yields": "8 servings",
   "description": "These delicious four cheese pizzas are bursting with flavor, and ready in under one hour.",
-  "total_time": 40.0,
-  "cook_time": 10.0,
-  "prep_time": 15.0,
+  "total_time": 40,
+  "cook_time": 10,
+  "prep_time": 15,
   "cuisine": "Italian Inspired",
   "ratings": 4.8,
   "ratings_count": 489,
   "nutrients": {
     "calories": "551 kcal",
     "carbohydrateContent": "54 g",
     "cholesterolContent": "58 mg",
-    "fatContent": "26 g",
     "fiberContent": "3 g",
     "proteinContent": "29 g",
     "saturatedFatContent": "11 g",
     "sodiumContent": "1183 mg",
     "sugarContent": "5 g",
-    "unsaturatedFatContent": "0 g"
+    "unsaturatedFatContent": "0 g",
+    "fatContent": "26 g"
   },
```

Comment:
Hm.  This is making me wonder whether we should have a preferred key order for `nutrients` elements -- and if so, how could we make it straightforward to write scrapers that adhere to that?  (a plugin, perhaps?)

Reply:
I think that could be beneficial but am not to sure on how it could work. 

Could this be another case for a custom test to be added with an associated script like the json check? Not sure how much we want to lean into that as a fixup tool.